pkg/util/networking/http.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package networking
```

Comment:
Aha, seems not too much conflicts, I wrote a shell to help me refactor this, gonna update soon :)